crates/swc_ecma_transforms_module/tests/fixture/common/issue-4864/output.cts

```diff
@@ -0,0 +1,98 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", {
+    value: true
+});
+function _export(target, all) {
+    for(var name in all)Object.defineProperty(target, name, {
+        enumerable: true,
+        get: all[name]
+    });
+}
+_export(exports, {
+    SETs: ()=>SETs,
+    SI: ()=>_ss.SI,
+    SSERT: ()=>_st().S_E_R_T,
+    ILIAN: ()=>_st().ILIAN,
+    IWI: ()=>_st().IWI,
+    SM: ()=>_st().SM,
+    VP: ()=>_st().VP,
+    SC: ()=>_sc().SC,
+    cSEP: ()=>_s().cSEP,
+    iCSEPFT: ()=>_s().iCSEPFT,
+    SASC: ()=>_s().SASC,
+    us: ()=>us,
+    C: ()=>C
+});
+const _ss = require("./path/to/SS");
```

Comment:
It's ok to leave it here.
This should be strip in TypeScript pass, but we do not have it in our swc_ecma_transforms_module test pass.